### ISSUE TITLE
WooCommerce integration: when affiliate selection is required still shows the word Optional

### ISSUE THREAD

**Issue by gspice** [AffiliateWP/affiliatewp-checkout-referrals#19]
First reported on this HS ticket:
https://secure.helpscout.net/conversation/597080051/83793/?folderId=1170842

AffiliateWP: 2.2.2
Checkout Referrals: 1.0.6
Integration:  WooCommerce

When the **Required Affiliate Selection** is checked under _Affiliates -> Settings -> Integrations_, the word `Optional` should not be displayed on the WC checkout page.  This does _not_ occur on the EDD or RCP integrations.  Please remove the word `Optional` if the affiliate selection is required on the WC integration.

Screenshot:

![woo-checkout-referrals-req](https://user-images.githubusercontent.com/3018245/41171626-175ef624-6b06-11e8-82ab-41e88eff12e2.png)

Current workaround for the customer was to hide via CSS:

```
.woocommerce form .form-row .optional {
	visibility: hidden;
}
```

**Comment by gspice** [comment]
Another came across this:
https://secure.helpscout.net/conversation/711923972/94559/

**Comment by gspice** [comment]
Another ticket on this:
https://secure.helpscout.net/conversation/729605503/96536?folderId=1170842

**Comment by michaelbeil** [comment]
Another ticket: https://secure.helpscout.net/conversation/817915393/0/?folderId=205524.

**Comment by gspice** [comment]
Amother occurance of this in HS:
https://secure.helpscout.net/conversation/1051595067/141833?folderId=1170842

>The addition "(optional)" appears on the Checkout page in the field description, although we have marked the selection field as a mandatory field (the red dot also appears).

**Comment by michaelbeil** [comment]
Another [ticket](https://secure.helpscout.net/conversation/1150377750/152396?folderId=205524) experiencing this:
>in Checkout Referrals settings I check the "Require Affiliate Selection" box (customer must select from the list option).
>However on Woocommerce check out page, that selection field label still says "optional", although it also shows the red asterisk and functions as 'required' (does not allow to place order without selecting the affiliate)

>when I switch to - customer enters affiliate ID - the 'optional' label disappears.

**Comment by paninapress** [comment]
PR: #46

**Comment by DrewAPicture** [comment]
Merged #46 against `release/1.0.9`
